kotlinx-coroutines-core/jvm/src/scheduling/CoroutineScheduler.kt

```diff
@@ -372,25 +372,34 @@ internal class CoroutineScheduler(
      * Dispatches execution of a runnable [block] with a hint to a scheduler whether
      * this [block] may execute blocking operations (IO, system calls, locking primitives etc.)
      *
-     * @param taskContext concurrency context of given [block]
-     * @param fair whether the task should be dispatched fairly (strict FIFO) or not (semi-FIFO)
+     * [taskContext] -- concurrency context of given [block].
+     * [tailDispatch] -- whether this [dispatch] call is the last action the (presumably) worker thread does in its current task.
```

Comment:
Just curious, why move away from using param tags?

Reply:
Lines are shorter :) 
For non-public API I personally find it more readable

Reply:
👍 It also gives more freedom in the layout of the doc. You can have parameters documented in the course of the prose.